_posts/en/newsletters/2019-07-31-newsletter.md

```diff
@@ -0,0 +1,246 @@
+  week's newsletter][Newsletter #56], we mentioned a [merged PR][Bitcoin
+  Core #16152] that disabled bloom filters in Bitcoin Core's default
+  configuration.  The author of the PR [announced][bloom announce] this
+  unreleased change to the Bitcoin-Dev mailing list and several people
+  replied with questions or concerns.  Some takeaways from the
+  discussion include:
+
+    - *No urgent action required:* as of this writing, over 20% of nodes
+      accepting incoming connections (listening nodes) are running a
+      version of Bitcoin Core that's over a year old [according to
+      BitNodes][bitnodes dashboard].  At least 5% are running a version
+      that's over two years old.  Extrapolating to the future, that
+      means there will still be over 500 listening nodes providing
+      bloom filters at the beginning of 2022.  So, even with no further
+      action, wallet developers may have a significant amount of time in
+      which to adapt their programs.
+
+    - *Spies likely to run their own nodes:* additional to non-upgraded
+      nodes, it's possible that block chain analysis companies will
+      continue to run their own nodes providing bloom filter support in
+      the future so that they can collect statistics from wallets whose
+      filters [leak information][filter privacy] about what addresses
+      they contain.
+
+    - *DNS seeds can return only nodes signaling BIP111:* most P2P light
+      wallets query one or more Bitcoin DNS seeds for a list of nodes to
+      use.  Some seeds allow filtering which nodes they return by what
+      service bits that node has configured[^dns-query], with the
+      service bit for bloom filter support being specified by
+      [BIP111][].
+
+    - *BIP157 would use more bandwidth than BIP37:* it was suggested
+      that some users of [BIP37][] bloom filters will be unable to
+      switch to [BIP157][] compact block filters because of the later's
+      higher bandwidth usage.  Optech briefly investigated filter size
+      in [Newsletter #43][] and found the filters themselves to use less
+      than 100 MB a month.  Clients would also need to download matching
+      blocks, which would vary depending on their activity (roughly one
+      block for every sent or received transaction for typical users; most
+      blocks are less than 2 MB in size).  It's unclear how many users
+      would be significantly affected by having to download that amount
+      of data.
+
+    - *Interested parties can run their own nodes:* the change to
+      Bitcoin Core only disables serving bloom filters by default.  It
+      doesn't remove the feature.  Authors of wallets depending on bloom
+      filter support can easily operate their own nodes, and they can
+      also try to persuade other node operators who aren't worried about
+      the DoS vulnerability to set the configuration option to true,
+      e.g.  `peerbloomfilters=1`.
```

Comment:
nit: extra space after "e.g.", perhaps would not be visible

Reply:
Agree it's an unnecessary space, but it shouldn't be visible, so shrug.